generators/entity/prompts.js

```diff
@@ -372,25 +372,25 @@ function askForTableName() {
             choices: [
                 {
                     value: 'no',
-                    name: 'No, the REST controller should use the repository directly'
+                    name: 'No, the REST controller should use the repository directly',
                 },
                 {
                     value: 'serviceClass',
-                    name: 'Yes, generate a separate service class'
+                    name: 'Yes, generate a separate service class',
                 },
                 {
                     value: 'serviceImpl',
-                    name: 'Yes, generate a separate service interface and implementation'
-                }
+                    name: 'Yes, generate a separate service interface and implementation',
+                },
```

Comment:
I think we can always generate interface and implementation

Reply:
@nicolas63 So I should remove the other option?

Reply:
Yes